src/modules/facility-covenant/dto/get-facility-covenants-response.dto.ts

```diff
@@ -0,0 +1,69 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { ENUMS } from '@ukef/constants';
+import { DateOnlyString } from '@ukef/helpers/date-only-string.type';
+import { IsEnum } from 'class-validator';
+
+export class GetFacilityCovenantsResponseDto {
+  @ApiProperty({
+    readOnly: true,
+    description:
+      'The identifier of the covenant in ACBS. When creating a covenant an identifier from the number generator should be used (if this value is skipped it will be auto-generated by ACBS without checking if it is already used).',
+  })
+  covenantIdentifier: string;
+
+  @ApiProperty({
+    readOnly: true,
+    description:
+      'The covenant type code: 43 for a UK Contract Value covenant, 46 for a Chargable Amount covenant, and 47 for a chargable Amount covenant not in GBP.',
```

Comment:
"...and 47 for a chargable Amount covenant not in GBP" - is this part accurate? The description on Anypoint said "47 for Chargable Amount then not GBP" but I wasn't sure what this meant @avaitonis

Reply:
I am not very familiar with covenant types, but new description looks good to me.

Reply:
Cool, I'll leave it as is then.